action/rlp_tx_test.go

```diff
@@ -599,7 +615,14 @@ func TestEthTxDecodeVerify(t *testing.T) {
 				require.NoError(err)
 				// verify against original tx
 				require.Equal(v.nonce, evmTx.Nonce())
-				require.Equal(v.price, evmTx.GasPrice().String())
+				price := MustBeTrueV(new(big.Int).SetString(v.price, 10))
+				require.Equal(price, evmTx.GasPrice())
+				require.Equal(price, evmTx.GasFeeCap())
+				if v.encoding == iotextypes.Encoding_ETHEREUM_BLOB {
+					require.Equal(price.Rsh(price, 1), evmTx.GasTipCap())
```

Comment:
should check blob fields

Reply:
it is covered by tests in `blob_tx_test.go`
also if the fields are wrong, the transaction hash will not match